Dockerfile.compute-node

```diff
@@ -337,11 +338,12 @@ USER root
 
 FROM rust-extensions-build AS pg-jsonschema-pg-build
 
-RUN git clone --depth=1 --single-branch --branch neon_abi_v0.1.4 https://github.com/vadim2404/pg_jsonschema/ && \
+RUN git clone --single-branch https://github.com/supabase/pg_jsonschema && \
     cd pg_jsonschema && \
+    # there is no release tag yet, but we need it due to the superuser fix in the control file
+    git checkout caeab60 && \
+    sed -i 's/pgx = "0.7.1"/pgx = { version = "0.7.3", features = [ "unsafe-postgres" ] }/g' Cargo.toml && \
```

Comment:
For `pg_graphql`, you also replaced the version for `pgx-tests`. Why don't you follow the same logic here?

Reply:
`pg_graphql` sets version as "=0.7.1" where equality fixes the minor revision. `pg_jsonschema` didn't do that so 0.7.3 satisfies "0.7.1" version spec as being semver-compatible.